kubespawner/spawner.py

```diff
@@ -781,6 +783,46 @@ def _hub_connect_port_default(self):
         """
     )
 
+    form_template = Unicode(
+        """<label for="profile">Please select a profile for your Jupyter:</label>
+        <select class="form-control" name="profile" required autofocus>
+        {input_template}
```

Comment:
call this `{inputs}` since it is not input_template, but the result of input_template being rendered multiple times.

Reply:
done